core/templates/dev/head/domain/exploration/EditableExplorationBackendApiService.js

```diff
@@ -113,8 +93,36 @@ oppia.factory('EditableExplorationBackendApiService', [
       });
     };
 
+    var _getExplorationUrl = function(explorationId, applyDraft, role) {
+      if (role) {
+        return UrlInterpolationService.interpolateUrl(
+        EDITABLE_EXPLORATION_DATA_ROLE_URL_TEMPLATE, {
+          exploration_id: explorationId,
+          role: JSON.stringify(role)
+        });
+      }
+      if (applyDraft) {
+        return UrlInterpolationService.interpolateUrl(
+        EDITABLE_EXPLORATION_DATA_DRAFT_URL_TEMPLATE, {
+          exploration_id: explorationId,
+          applyDraft: JSON.stringify(applyDraft)
+        });
+      }
+      return UrlInterpolationService.interpolateUrl(
+        EDITABLE_EXPLORATION_DATA_URL_TEMPLATE, {
+          exploration_id: explorationId
+        }
+      );
+    };
+
     return {
-      fetchExploration: function(explorationId, applyDraft) {
+      fetchExploration: function(explorationId) {
+        return $q(function(resolve, reject) {
+          _fetchExploration(explorationId, null, resolve, reject);
+        });
+      },
+
+      fetchApplyDraftExploration: function(explorationId, applyDraft) {
```

Comment:
That's a good question and I haven't thought about it yet. Given the nature of these backend api services, we're sort of moving toward a 1:1 relationship between backend URLs and frontend api services, so I lean slightly toward a separate ``EditableExplorationDraftBackendApiService`` in the future.